cmd/fuzzcrypto/main.go

```diff
@@ -0,0 +1,153 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"path"
+	"path/filepath"
+	"regexp"
+	"strconv"
+	"strings"
+	"time"
+)
+
+const description = `
+fuzzcrypto runs a curated list of fuzz tests sequentially.
+
+Example that runs all the available fuzz tests:
+
+  go run .
+
+Example that runs only the go-cose subset for 1 minute:
+
+  go run . -run go-cose -fuzztime 1m
+
+The total fuzzing time, set by -fuzztime, is distributed among the executed fuzz tests.
+fuzzcrypto might run longer than -fuzztime as it does not take into account build time
+nor the fuzz seeding time.
+`
+
+func main() {
+	var verbose = flag.Bool("v", false, "verbose logging")
+	var fuzzDuration durationOrCountFlag
+	flag.Var(&fuzzDuration, "fuzztime", "total time to spend fuzzing or number of iterations that the fuzz target will be executed before exiting; default is to run for 5min")
```

Comment:
I think the `-help` doc should mention the syntax, maybe just copy upstream's (with our own default) now that we use the same syntax?

```
-fuzztime t
    Run enough iterations of the fuzz target during fuzzing to take t,
    specified as a time.Duration (for example, -fuzztime 1h30s).
	The default is to run forever.
    The special syntax Nx means to run the fuzz target N times
    (for example, -fuzztime 1000x).
```

Reply:
Done!